interface/drugs/add_edit_drug.php

```diff
@@ -363,8 +363,9 @@ function sel_related() {
     'related_code' => '',
     );
 }
+$title = $drug_id ? "Update Drug" : "Add Drug";
```

Comment:
for translations, much better to translate the string so it's picked up in the translation pipeline collection script(s):
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/README

so would do:
```
$title = $drug_id ? xl("Update Drug") : xl("Add Drug");
```
And then escape them below via text() call